paste/httpserver.py

```diff
@@ -1355,7 +1355,7 @@ def server_runner(wsgi_app, global_conf, **kwargs):
         if name in kwargs:
             kwargs[name] = asbool(kwargs[name])
     threadpool_options = {}
-    for name, value in kwargs.items():
+    for name, value in kwargs.copy().items():
```

Comment:
`2to3` suggests `for name, value in list(kwargs.items()):` , not sure if there is any practical difference.

Reply:
I guess not, I suppose 2to3 does this in case you'd do
```
In [1]: d = {'a': 'b'}

In [2]: items = d.items()

In [3]: items[0]
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-3-5dbb89d7bbf9> in <module>()
----> 1 items[0]

TypeError: 'dict_items' object does not support indexing

In [4]:
```
while that would have worked on python2

Reply:
actually that is a little nicer, I went with casting to list.